dnn/data/converters/roco.py

```diff
@@ -9,6 +9,13 @@
     https://uillinoisedu-my.sharepoint.com/:u:/g/personal/yixiaos3_illinois_edu/Ef5LIWMYRpNLgMuXGHOrAZoBVKdsymnJ8xWinbJO_pX9OQ?e=POF0Ts
 """
 
+import sys
+import os
+sys.path.insert(0, os.path.abspath('.').split('/dnn')[0])
+print('Trim path to',sys.path[0])
```

Comment:
This should be done in a virtual environment, by `export PYTHONPATH=<path to irm_Autonomy_2020>:$PYTHONPATH`.